composer.json

```diff
@@ -119,7 +119,10 @@
         "egulias/email-validator": "~1.2,>=1.2.8|~2.0",
         "symfony/phpunit-bridge": "^3.4.31|^4.3.4|~5.0",
         "symfony/security-acl": "~2.8|~3.0",
-        "phpdocumentor/reflection-docblock": "^3.0|^4.0"
+        "phpdocumentor/reflection-docblock": "^3.0|^4.0",
+        "twig/cssinliner-extra": "^2.12",
+        "twig/inky-extra": "^2.12",
+        "twig/markdown-extra": "^2.12"
```

Comment:
wouldn't this forbid testing against Twig 1.x (while at runtime, Symfony will still allow it) ? These extensions don't support Twig 1

Reply:
I know but I don't think there is any other way.

Reply:
TBH I don’t think it’s worth to drop Twig 1 support and add 3 new dependencies “just” for a notification email.
This is just my POV, please don’t feel offended 🤝

What is the reason not to wait until the ne notifier component gets released?

Reply:
We don't add 3 deps (we are talking about dev requirements here).

I don't see how waiting for the Notifier component will change anything. The 2 features are independant and the code here won't change when the notifier lands.

Reply:
And to be clear, we don't drop Twig 1.x support. We are just not able to run the tests for it in our CI anymore (but as the tests will run in 4.3, I don't think this is a major issue -- and for master, 1.x is dropped anyway).

Reply:
In a env where we are using twig 1.x we can drop theses 3 deps and skip related tests

Reply:
env: you mean in the CI? Again, I don't  think this is worth it as Twig 1.x is tested on 4.3 and master does not support 1.x.

Reply:
Sorry I missed that these are dev dependencies. 👍🏻

Reply:
Yes the CI. 
Anyway I'm okay with your proposal.